build-system/runner/src/org/ampproject/AmpPass.java

```diff
@@ -74,9 +74,21 @@ public AmpPass(AbstractCompiler compiler, boolean isProd, Set<String> stripTypeS
     } else if (isProd && isFunctionInvokeAndPropAccess(n, "$mode.getMode",
         ImmutableSet.of("minified"))) {
       replaceWithBooleanExpression(true, n, parent);
+    } else if (isProd && isLValueName(n, "OVERRIDE_IS_NOT_PROD")) {
```

Comment:
done. added immutablemap coming from the runner passing it into the custom pass.